_episodes/01-intro.md

```diff
@@ -101,7 +101,7 @@ Documents   Library     Music       Public
 > ~~~
 > {: .output}
 >
-> Usually this means that you have mis-typed the command.
+> Usually this means that you have mis-typed the command. You get the same error, when you type the tool command but the tool has not been installed.
```

Comment:
Hi, I don't really like the term "tool".

Perhaps reword to "You can get the same error if the command you want is not installed"

Reply:
Hi, Thanks for the suggestion. I will edit it.

Reply:
To add my 2p:
Sure it's *programs* which are installed? They have command line interfaces, or graphical user interfaces, but the command is what you issue from the shell in order to run the program. So you don't install a *command*. Or have I missed something?